org.speed.dreams.yaml

```diff
@@ -0,0 +1,151 @@
+        
+  - name: openscenegraph
+    buildsystem: cmake-ninja
+    config-opts:
+    - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: archive
+        url: https://github.com/openscenegraph/OpenSceneGraph/archive/OpenSceneGraph-3.4.1.tar.gz
+        sha256: 930eb46f05781a76883ec16c5f49cfb29a059421db131005d75bec4d78401fd5
+  
+  - name: libxmu
+    sources:
+      - type: archive
+        url: https://www.x.org/archive//individual/lib/libXmu-1.1.2.tar.bz2
+        sha256: 756edc7c383254eef8b4e1b733c3bf1dc061b523c9f9833ac7058378b8349d0b
+  
+  - name: plib
+    rm-configure: true
+    config-opts:
+      - CXXFLAGS=-fPIC
+    sources:
+      - type: archive
+        url: http://plib.sourceforge.net/dist/plib-1.8.5.tar.gz
+        sha1: c2cf7e3e1e58f7b63dae4bb21e4fa82c3e4d4cfc
+        
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+
+  - name: enet
+    sources:
+      - type: archive
+        url: http://enet.bespin.org/download/enet-1.3.13.tar.gz
+        sha256: 82501c0248330a3a6bcac907907760e59b18cb662ebb42fe2c91e1557f1bc8ab452bf3fe7477dc01fa7a6e3b6124e1eb8eac46382d5bbebb2394879f9568b5df
+  
+  - name: freealut
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://github.com/vancegroup/freealut/archive/freealut_1_1_0.tar.gz
+        sha256: c3880d7dbb90c7db54e1d6b88aa0a34dd9e2d828fd389be0d2cbb2632b0885dd
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+          
+  - shared-modules/udev/udev-175.json
+  
+  - name: speed-dreams-2
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DSD_BINDIR=/app/bin
+      - -DSD_DATADIR=/app/share/speed-dreams-2
+      - -DSD_LIBDIR=/app/lib/speed-dreams-2
+      - -DOPTION_OFFICIAL_ONLY=ON
+    
+    sources:
+      - type: file
+        url: https://downloads.sourceforge.net/sourceforge/speed-dreams/speed-dreams-src-base-2.2.2-rc2-r6527.tar.xz
+        md5: 956bda0acbcb89eaa2ca742dbedc8941
+      
+      - type: file
+        url: https://downloads.sourceforge.net/sourceforge/speed-dreams/speed-dreams-src-wip-cars-and-tracks-2.2.2-rc2-r6527.tar.xz
+        md5: bc183b3e80c742bc249f630cb029920c
+          
+      - type: file
+        url: https://downloads.sourceforge.net/sourceforge/speed-dreams/speed-dreams-src-hq-cars-and-tracks-2.2.2-rc2-r6527.tar.xz
+        md5: ed11647ad2d479dc4012ffd79375b995
+          
+      - type: file
+        url: https://downloads.sourceforge.net/sourceforge/speed-dreams/speed-dreams-src-more-hq-cars-and-tracks-2.2.2-rc2-r6527.tar.xz
+        md5: d615eda2c443a2e5ce74cf45d5374062
+        
+      - type: file
+        path: org.speed_dreams.SpeedDreams.desktop
+        
+      - type: file
+        path: org.speed_dreams.SpeedDreams.appdata.xml
+
+      - type: shell
+        commands:
+          - tar xf speed-dreams-src-base-2.2.2-rc2-r6527.tar.xz
```

Comment:
All source types take `dest` argument. Why not use it instead?

Reply:
I try with many dests, but don't work